Runtime/Encoding.cs

```diff
@@ -12,7 +12,7 @@ public static class SerializationConstants {
   /// 
   /// Defaults to 65535. Can be changed.
   /// </summary>
-  public static int kMaxMessageSize = 65535;
```

Comment:
HouraiSerialization is meant to be generic and work with any UDP transport layer. I'd suggest reverting this change.